extensions/objects/templates/FilepathEditor.js

```diff
@@ -217,10 +222,10 @@ oppia.directive('filepathEditor', [
           var ctx = canvas.getContext('2d');
           ctx.drawImage(img, 0, 0, dimensions.width, dimensions.height);
 
-          // Return a File object obtained from the data in the canvas.
-          var file = $scope.data.metadata.uploadedFile;
-          var blob = dataURIToBlob(canvas.toDataURL(file.type, 1));
-
+          // We export to PNG because not all browsers support exports to all
+          // possible file types (like gif).
+          var blob = dataURIToBlob(
+            canvas.toDataURL('image/' + RESAMPLED_IMAGE_FORMAT, 1));
```

Comment:
Yes, I did -- I uploaded a jpg file through the UI, and I did the same for a png file. Everything gets transformed to png anyway so I think these are the only possibilities.